### ISSUE TITLE
give admins the possibility to do some stealth action.

### ISSUE THREAD

**Issue by jxsl13** [teeworlds/teeworlds#2225]
Hiding, who they watch + hiding that they are admin.

**Comment by fokkonaut** [comment]
How can be seen that they are admins?

**Comment by jxsl13** [comment]
Looking at people who are votable + the gamer client for example clearly shows that with an asterisks/star behind the name.

**Comment by Dune-jr** [comment]
You don't need any modification, people that you cannot vote-kick in the menus are logged in rcon.
In 0.6, you couldn't votekick them either though (although you had to actually attempt to votekick).

Like I told you earlier, I would argue that
- if you want to do "stealth action", you can just not log in
- people who try *this* hard to dodge admins probably know the name of the admins anyway
- server modifications for edge use cases (it's a one-liner)

**Comment by H-M-H** [comment]
If you want to be really stealthy, you could use the external console: https://github.com/teeworlds/teeworlds/blob/96473b0450d58bdfdb8f5b990f1799e905babd40/src/engine/shared/config_variables.h#L95-L100
Like that you do not even need to be playing on the server.

**Comment by oy** [comment]
If you want to be stealthy you log in just when you have to, do your thing and then log of again.
Don't think there needs to be a special feature for that.

**Comment by jxsl13** [comment]
I wonder, has anyone of you done any admin work during 0.6?

**Comment by Sonix-** [comment]
> I wonder, has anyone of you done any admin work during 0.6?

Yes, I log in, I ban  and that's all

Else @H-M-H 's suggestion is also reasonable

**Comment by jxsl13** [comment]
I'd say it's reasonable, but not practicable(switching between multiple application sucks). Like most quality of life features contained in the client, they are not needed, but highly appreciated by the people who use them. Especially because you cannot do that directly from within the client(econ). Then I'd ask to add a way to connect via econ directly within the client. 
I would guess that the purpose of the econ was to do stuff, while not being on the server, where you can clearly not detect cheaters, when only using econ.

**Comment by Dune-jr** [comment]
I believe you that this scenario is a real possibility. However, it seems like the large majority of server admins are open about their identity, and there are already tools at your disposal to cover this edge case (external console, or just don't log in). 
It constraints you to use a fake identity anyway so you're not going to be doing this all the time.


Now, this seems to be very important for you, so you could also simply comment out this line on your server:
https://github.com/teeworlds/teeworlds/blob/d9378b3c9b2049154646c04380e88bafbacf2dce/src/game/server/player.cpp#L147-L148

**Comment by sepulzera** [comment]
Please note that it is not allowed to run a server with any modification as vanilla gametype. (And I honestly would like to have that rule changed)

**Comment by jxsl13** [comment]
It's not solely my problem, because I mostly play on zCatch it's easily modifiable. I think this is something that affects the general public and admins that want or need to hide their identities. I opened this issue, because I saw a person(not me) who had exactly that problem on ddrace and thought that it might make sense to address that issue on the teeworlds github. What's rather disappointing is that admins do not necessarily look into the development process and thus don't comment to tell whether they would or want to use such a feature.
I think it would make most sense to have some kinda hidden ctrl+F2 console for econ(connecting to current server ip or connecting to custom ip), in order to utilize h-m-h's approach without modifying the rcon logic too much.
